modules/priceFloors.js

```diff
@@ -190,6 +192,7 @@ export function getFloor(requestParams = {currency: 'USD', mediaType: '*', size:
   let bidRequest = this;
   let floorData = _floorDataForAuction[bidRequest.auctionId];
   if (!floorData || floorData.skipped) return {};
+  if (floorData.hasOwnProperty('floorMin')) floorData.data.floorMin = floorData.floorMin;
```

Comment:
This is going to run each time getFloor is called. Which is probably not ideal.

Better to just set `floorData.data.floorMin` once.

I would recommend doing this in `createFloorsDataForAuction` which will initialize the entire data object, then you will just be able to access it where needed.